compiler/src/iree/compiler/Codegen/SPIRV/test/pipeline_matvec.mlir

```diff
@@ -211,7 +223,7 @@ hal.executable @i4_dequant_matvec_f16_subgroup_64 {
 //         CHECK:   %[[ACCESS:.+]] = spirv.AccessChain %[[RADDR]][{{.*}}, %[[OFFSET]]] : !spirv.ptr<!spirv.struct<(!spirv.rtarray<i32, stride=4> [0])>, StorageBuffer>, i32, i32
 //         CHECK:   spirv.Load "StorageBuffer" %[[ACCESS]] : i32
 
-//         CHECK:   spirv.VectorShuffle [0 : i32, 0 : i32, 1 : i32, 1 : i32] %{{.*}} : vector<2xi32>, %106 : vector<2xi32> -> vector<4xi32>
+//         CHECK:   spirv.VectorShuffle [0 : i32, 0 : i32, 1 : i32, 1 : i32] %{{.*}} : vector<2xi32>, {{.*}} : vector<2xi32> -> vector<4xi32>
```

Comment:
Ha, the assembly seems off--we need to fix the `spirv.VectorShuffle` assembly.

Reply:
what's off here? it seems fine to me modulo the hardcoded value

Reply:
Oh I see -- the types interleaved with names

Reply:
Yeah. Nothing to do for this pull request--just something we need to fix upstream.